app/src/main/java/org/openimis/imispolicies/CummulativeIndicators.java

```diff
@@ -166,6 +170,7 @@ public void getCommulativeIndicators(String DateFrom, String DateTo){
                 public void run() {
                     if(commulative.length() != 0){
                         try {
+                            //TODO:  Check if iteration was better
```

Comment:
Is it still a TODO ?

Reply:
No, iteration was used in soap call, comment removed